README.md

```diff
@@ -452,6 +491,18 @@ The `session` argument should be a session if found, otherwise `null` or
 `undefined` if the session was not found (and there was no error). A special
 case is made when `error.code === 'ENOENT'` to act like `callback(null, null)`.
 
+### store.load(sid, callback)
+
+**Required** (available from `express-session`)
+
+This required method is used to load a `Session` instance via the given session ID (`sid`). The `callback` should be called as `callback(error, session)`. This method calls `store.createSession` internally.
```

Comment:
Addessed in https://github.com/expressjs/session/pull/354/commits/5fbf848ba15048fb4f36ea525a6e086ffc62b186